dss-api.yml

```diff
@@ -345,6 +345,14 @@ paths:
                 pattern: "[A-Za-z0-9]{8}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{12}"
             required:
               - uuid
+        500:
+          description: Internal Server Error
+          schema:
+            $ref: '#/definitions/Error'
```

Comment:
Think I misunderstood. You want all of the error types to look like this:

```        default:
          description: Unexpected error
          schema:
            allOf:
              - $ref: '#/definitions/Error'
              - type: object
                properties:
                  code:
                    type: string
                    description: Machine-readable error code.  The types of return values should not be changed lightly.
                    enum: [unhandled_exception, not_found]
                required:
                  - code```
Where enum lists all of the possible error types? So there will just be one error listing for each endpoint?